### ISSUE TITLE
GameServices.google.auth.getAuth() ?

### ISSUE THREAD

**Issue by MyMyGames** [myflashlab/GameServices-ANE#15]
Why there is no response at all after login when calling authorization's function "GameServices.google.auth.getAuth();" ?
I listening to GameServices.google.auth.addEventListener(AuthEvents.AUTH, onAuthDataRetrived);

```
private function onLoginSuccess(e:AuthEvents):void
{ 
 trace("onLoginSuccess"); 
 GameServices.google.auth.addEventListener(AuthEvents.AUTH, onAuthDataRetrived); 
 GameServices.google.auth.getAuth(); 
}

```
^^^^^^^
it's ok!

```
private function onAuthDataRetrived(e:AuthEvents):void
{ 
  trace("name = " + e.auth.name); 
  trace("email = " + e.auth.email); 
  trace("scopes = " + e.auth.scopes); 
  trace("token = " + e.auth.token); 
}
```
^^^^^^^
no reaction (

**Comment by myflashlab** [comment]
Are you using the latest version of the ANE? does [this comment](https://github.com/myflashlab/GameServices-ANE/issues/14#issuecomment-297029182) help?

**Comment by MyMyGames** [comment]
Latest version 2.2.1 from MyFlashLabClub_ANEs_package.
No it does not help.

**Comment by myflashlab** [comment]
Please give us more information. Try to connect your device to your computer while checking the logcat window on AndroidStudio. see if any error message is shown?

**Comment by MyMyGames** [comment]
After first login, on next logins 
if "canDoSilentLogin == true." No event  AuthEvents.LOGIN

> 05-10 14:50:25.278: D/DefContainer(19241): Copying /data/local/tmp/RainbowSquad.apk to base.apk
05-10 15:07:08.095: W/ActivityManager(609): Duplicate finish request for ActivityRecord{d163ca7 u0 air.com.RainbowSquad/.AppEntry t18461 f}
05-10 15:07:08.422: I/WindowState(609): WIN DEATH: Window{7908f6d u0 air.com.RainbowSquad/air.com.RainbowSquad.AppEntry}
05-10 15:07:08.556: I/ActivityManager(609): Process air.com.RainbowSquad (pid 23973) has died
05-10 15:11:23.575: I/ActivityManager(609): Killing 23242:air.com.RainbowSquad/u10a363 (adj 15): empty #17
05-10 15:11:29.244: I/ActivityManager(609): Killing 23940:air.com.RainbowSquad/u11a363 (adj 15): empty #17
05-10 15:12:22.212: D/DefContainer(25685): Copying /data/local/tmp/RainbowSquad.apk to base.apk
05-10 15:12:39.992: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=-1: uninstall pkg
05-10 15:12:40.711: I/PackageManager(609): Package air.com.RainbowSquad codePath changed from /data/app/air.com.RainbowSquad-1 to /data/app/air.com.RainbowSquad-2; Retaining data and using new
05-10 15:12:40.714: W/PackageManager(609): Code path for air.com.RainbowSquad changing from /data/app/air.com.RainbowSquad-1 to /data/app/air.com.RainbowSquad-2
05-10 15:12:40.714: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=-1: replace pkg
05-10 15:12:40.714: W/PackageManager(609): Resource path for air.com.RainbowSquad changing from /data/app/air.com.RainbowSquad-1 to /data/app/air.com.RainbowSquad-2
05-10 15:12:41.397: W/PackageManager(609): Unknown permission com.sec.android.provider.badge.permission.READ in package air.com.RainbowSquad
05-10 15:12:41.397: W/PackageManager(609): Unknown permission com.sec.android.provider.badge.permission.WRITE in package air.com.RainbowSquad
05-10 15:12:41.397: W/PackageManager(609): Unknown permission com.htc.launcher.permission.READ_SETTINGS in package air.com.RainbowSquad
05-10 15:12:41.397: W/PackageManager(609): Unknown permission com.htc.launcher.permission.UPDATE_SHORTCUT in package air.com.RainbowSquad
05-10 15:12:41.397: W/PackageManager(609): Unknown permission com.sonyericsson.home.permission.BROADCAST_BADGE in package air.com.RainbowSquad
05-10 15:12:41.397: W/PackageManager(609): Unknown permission com.anddoes.launcher.permission.UPDATE_COUNT in package air.com.RainbowSquad
05-10 15:12:41.397: W/PackageManager(609): Unknown permission com.majeur.launcher.permission.UPDATE_BADGE in package air.com.RainbowSquad
05-10 15:12:42.348: W/PackageManager(609): Couldn't remove dex file for package:  at location /data/app/air.com.RainbowSquad-1/base.apk, retcode=-1
05-10 15:12:42.348: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=0: pkg removed
05-10 15:12:42.451: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=10: pkg removed
05-10 15:12:42.475: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=11: pkg removed
05-10 15:12:42.530: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.PACKAGE_REMOVED dat=package:air.com.RainbowSquad flg=0x4000010 (has extras) }: user 12 is stopped
05-10 15:12:42.568: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.PACKAGE_ADDED dat=package:air.com.RainbowSquad flg=0x4000010 (has extras) }: user 12 is stopped
05-10 15:12:42.579: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.PACKAGE_REPLACED dat=package:air.com.RainbowSquad flg=0x4000010 (has extras) }: user 12 is stopped
05-10 15:12:42.604: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.MY_PACKAGE_REPLACED flg=0x4000010 pkg=air.com.RainbowSquad (has extras) }: user 12 is stopped
05-10 15:13:48.868: I/ActivityManager(609): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=air.com.RainbowSquad/.AppEntry bnds=[24,180][216,372] (has extras)} from uid 10013 on display 0
05-10 15:13:48.916: I/ActivityManager(609): Start proc 28339:air.com.RainbowSquad/u0a363 for activity air.com.RainbowSquad/.AppEntry
05-10 15:14:14.187: W/ActivityManager(609): Duplicate finish request for ActivityRecord{d16a913 u0 air.com.RainbowSquad/.AppEntry t18463 f}
05-10 15:14:14.436: I/WindowState(609): WIN DEATH: Window{1983975 u0 air.com.RainbowSquad/air.com.RainbowSquad.AppEntry}
05-10 15:14:14.596: I/ActivityManager(609): Process air.com.RainbowSquad (pid 28339) has died
05-10 15:19:53.193: D/DefContainer(31169): Copying /data/local/tmp/RainbowSquad.apk to base.apk
05-10 15:20:13.134: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=-1: uninstall pkg
05-10 15:20:13.972: I/PackageManager(609): Package air.com.RainbowSquad codePath changed from /data/app/air.com.RainbowSquad-2 to /data/app/air.com.RainbowSquad-1; Retaining data and using new
05-10 15:20:13.976: W/PackageManager(609): Code path for air.com.RainbowSquad changing from /data/app/air.com.RainbowSquad-2 to /data/app/air.com.RainbowSquad-1
05-10 15:20:13.976: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=-1: replace pkg
05-10 15:20:13.976: W/PackageManager(609): Resource path for air.com.RainbowSquad changing from /data/app/air.com.RainbowSquad-2 to /data/app/air.com.RainbowSquad-1
05-10 15:20:14.681: W/PackageManager(609): Unknown permission com.sec.android.provider.badge.permission.READ in package air.com.RainbowSquad
05-10 15:20:14.681: W/PackageManager(609): Unknown permission com.sec.android.provider.badge.permission.WRITE in package air.com.RainbowSquad
05-10 15:20:14.681: W/PackageManager(609): Unknown permission com.htc.launcher.permission.READ_SETTINGS in package air.com.RainbowSquad
05-10 15:20:14.682: W/PackageManager(609): Unknown permission com.htc.launcher.permission.UPDATE_SHORTCUT in package air.com.RainbowSquad
05-10 15:20:14.682: W/PackageManager(609): Unknown permission com.sonyericsson.home.permission.BROADCAST_BADGE in package air.com.RainbowSquad
05-10 15:20:14.682: W/PackageManager(609): Unknown permission com.anddoes.launcher.permission.UPDATE_COUNT in package air.com.RainbowSquad
05-10 15:20:14.682: W/PackageManager(609): Unknown permission com.majeur.launcher.permission.UPDATE_BADGE in package air.com.RainbowSquad
05-10 15:20:15.932: W/PackageManager(609): Couldn't remove dex file for package:  at location /data/app/air.com.RainbowSquad-2/base.apk, retcode=-1
05-10 15:20:15.932: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=0: pkg removed
05-10 15:20:16.026: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=10: pkg removed
05-10 15:20:16.033: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=11: pkg removed
05-10 15:20:16.096: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.PACKAGE_REMOVED dat=package:air.com.RainbowSquad flg=0x4000010 (has extras) }: user 12 is stopped
05-10 15:20:16.127: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.PACKAGE_ADDED dat=package:air.com.RainbowSquad flg=0x4000010 (has extras) }: user 12 is stopped
05-10 15:20:16.140: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.PACKAGE_REPLACED dat=package:air.com.RainbowSquad flg=0x4000010 (has extras) }: user 12 is stopped
05-10 15:20:16.141: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.MY_PACKAGE_REPLACED flg=0x4000010 pkg=air.com.RainbowSquad (has extras) }: user 12 is stopped
05-10 15:20:32.671: I/ActivityManager(609): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=air.com.RainbowSquad/.AppEntry bnds=[24,180][216,372] (has extras)} from uid 10013 on display 0
05-10 15:20:32.909: I/ActivityManager(609): Start proc 32259:air.com.RainbowSquad/u0a363 for activity air.com.RainbowSquad/.AppEntry
05-10 15:20:32.986: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 21 lines
05-10 15:20:34.951: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 3 lines
05-10 15:20:35.438: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 2 lines
05-10 15:20:36.954: I/Finsky(31419): [1] com.google.android.finsky.utils.PermissionPolicies$PermissionPolicyService.onStartCommand(18): post-install permissions check for air.com.RainbowSquad
05-10 15:20:38.684: I/ActivityManager(609): Displayed air.com.RainbowSquad/.AppEntry: +5s786ms (total +7s129ms)
05-10 15:20:39.649: D/Wear_Controller(31472): Received broadcast action=android.intent.action.PACKAGE_ADDED and uri=air.com.RainbowSquad
05-10 15:20:42.138: I/Finsky(31691): [1] com.google.android.finsky.utils.PermissionPolicies$PermissionPolicyService.onStartCommand(18): post-install permissions check for air.com.RainbowSquad
05-10 15:20:43.689: D/Wear_Controller(31803): Received broadcast action=android.intent.action.PACKAGE_ADDED and uri=air.com.RainbowSquad
05-10 15:20:44.971: I/Finsky(32704): [22305] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:20:45.013: I/Finsky(32704): [22332] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:20:45.069: I/Finsky(32704): [22304] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:20:45.073: I/Finsky(32704): [22305] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:20:45.452: I/Finsky(32704): [22332] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:20:45.454: I/Finsky(32704): [22304] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:20:45.472: I/Finsky(32704): [22305] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:20:45.474: I/Finsky(32704): [22332] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:20:46.252: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 36 lines
05-10 15:20:46.260: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 1 line
05-10 15:20:46.333: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 1 line
05-10 15:20:47.797: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 1 line
05-10 15:20:47.885: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 1 line
05-10 15:20:47.905: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 85 lines
05-10 15:20:48.010: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 1 line
05-10 15:20:48.126: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 1 line
05-10 15:20:48.837: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 1 line
05-10 15:20:51.976: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 3 lines
05-10 15:20:52.129: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 2 lines
05-10 15:20:52.330: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 3 lines
05-10 15:20:52.398: I/chatty(32259): uid=10363(air.com.RainbowSquad) expire 3 lines
05-10 15:20:57.500: W/ActivityManager(609): Duplicate finish request for ActivityRecord{3e4d71 u0 air.com.RainbowSquad/.AppEntry t18466 f}
05-10 15:20:57.679: I/WindowState(609): WIN DEATH: Window{d6adc45 u0 air.com.RainbowSquad/air.com.RainbowSquad.AppEntry}
05-10 15:20:57.836: I/ActivityManager(609): Process air.com.RainbowSquad (pid 32259) has died
05-10 15:21:13.629: I/ActivityManager(609): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=air.com.RainbowSquad/.AppEntry bnds=[24,180][216,372] (has extras)} from uid 10013 on display 0
05-10 15:21:13.655: I/ActivityManager(609): Start proc 1785:air.com.RainbowSquad/u0a363 for activity air.com.RainbowSquad/.AppEntry
05-10 15:21:13.765: I/FA(1785):   adb shell setprop debug.firebase.analytics.app air.com.RainbowSquad
05-10 15:21:16.351: I/ActivityManager(609): Displayed air.com.RainbowSquad/.AppEntry: +2s707ms
05-10 15:21:19.605: I/Finsky(32704): [22304] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:21:19.608: I/Finsky(32704): [22305] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:21:19.670: I/Finsky(32704): [22332] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:21:19.673: I/Finsky(32704): [22304] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:21:19.686: I/Finsky(32704): [22305] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:21:19.689: I/Finsky(32704): [22332] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:21:19.709: I/Finsky(32704): [22304] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:21:19.711: I/Finsky(32704): [22305] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:21:21.216: I/b(1785): ##### Serializing blacklist ad id to /data/user/0/air.com.RainbowSquad/cache/.chartboost/.adId
05-10 15:21:21.243: D/b(1785): ##### Before html injection file path /data/user/0/air.com.RainbowSquad/cache/.chartboost/html/237f487bc335be9186e597dcc7376a28479da189--1760380433
05-10 15:21:21.742: I/b(1785): ##### Serializing blacklist ad id to /data/user/0/air.com.RainbowSquad/cache/.chartboost/.adId
05-10 15:21:29.702: W/ActivityManager(609): Duplicate finish request for ActivityRecord{f0c6f2d u0 air.com.RainbowSquad/.AppEntry t18467 f}
05-10 15:21:29.915: I/WindowState(609): WIN DEATH: Window{a2e0de3 u0 air.com.RainbowSquad/air.com.RainbowSquad.AppEntry}
05-10 15:21:30.136: I/ActivityManager(609): Process air.com.RainbowSquad (pid 1785) has died
05-10 15:23:10.223: D/DefContainer(2387): Copying /data/local/tmp/RainbowSquad.apk to base.apk
05-10 15:23:15.664: I/PackageManager.DexOptimizer(609): Running dexopt (dex2oat) on: /data/app/vmdl919898372.tmp/base.apk pkg=air.com.RainbowSquad isa=arm vmSafeMode=false debuggable=false oatDir = /data/app/vmdl919898372.tmp/oat bootComplete=true
05-10 15:23:24.295: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=-1: uninstall pkg
05-10 15:23:24.957: I/PackageManager(609): Package air.com.RainbowSquad codePath changed from /data/app/air.com.RainbowSquad-1 to /data/app/air.com.RainbowSquad-2; Retaining data and using new
05-10 15:23:24.968: W/PackageManager(609): Code path for air.com.RainbowSquad changing from /data/app/air.com.RainbowSquad-1 to /data/app/air.com.RainbowSquad-2
05-10 15:23:24.968: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=-1: replace pkg
05-10 15:23:24.968: W/PackageManager(609): Resource path for air.com.RainbowSquad changing from /data/app/air.com.RainbowSquad-1 to /data/app/air.com.RainbowSquad-2
05-10 15:23:25.594: W/PackageManager(609): Unknown permission com.sec.android.provider.badge.permission.READ in package air.com.RainbowSquad
05-10 15:23:25.594: W/PackageManager(609): Unknown permission com.sec.android.provider.badge.permission.WRITE in package air.com.RainbowSquad
05-10 15:23:25.594: W/PackageManager(609): Unknown permission com.htc.launcher.permission.READ_SETTINGS in package air.com.RainbowSquad
05-10 15:23:25.594: W/PackageManager(609): Unknown permission com.htc.launcher.permission.UPDATE_SHORTCUT in package air.com.RainbowSquad
05-10 15:23:25.594: W/PackageManager(609): Unknown permission com.sonyericsson.home.permission.BROADCAST_BADGE in package air.com.RainbowSquad
05-10 15:23:25.594: W/PackageManager(609): Unknown permission com.anddoes.launcher.permission.UPDATE_COUNT in package air.com.RainbowSquad
05-10 15:23:25.594: W/PackageManager(609): Unknown permission com.majeur.launcher.permission.UPDATE_BADGE in package air.com.RainbowSquad
05-10 15:23:26.115: W/PackageManager(609): Couldn't remove dex file for package:  at location /data/app/air.com.RainbowSquad-1/base.apk, retcode=-1
05-10 15:23:26.116: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=0: pkg removed
05-10 15:23:26.241: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=10: pkg removed
05-10 15:23:26.255: I/ActivityManager(609): Force stopping air.com.RainbowSquad appid=10363 user=11: pkg removed
05-10 15:23:26.306: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.PACKAGE_REMOVED dat=package:air.com.RainbowSquad flg=0x4000010 (has extras) }: user 12 is stopped
05-10 15:23:26.344: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.PACKAGE_ADDED dat=package:air.com.RainbowSquad flg=0x4000010 (has extras) }: user 12 is stopped
05-10 15:23:26.397: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.PACKAGE_REPLACED dat=package:air.com.RainbowSquad flg=0x4000010 (has extras) }: user 12 is stopped
05-10 15:23:26.398: W/ActivityManager(609): Skipping broadcast of Intent { act=android.intent.action.MY_PACKAGE_REPLACED flg=0x4000010 pkg=air.com.RainbowSquad (has extras) }: user 12 is stopped
05-10 15:23:28.414: D/Wear_Controller(21277): Received broadcast action=android.intent.action.PACKAGE_REMOVED and uri=air.com.RainbowSquad
05-10 15:23:28.417: E/Drive.UninstallOperation(21277): Package still installed air.com.RainbowSquad
05-10 15:23:32.339: D/Wear_Controller(2728): Received broadcast action=android.intent.action.PACKAGE_REMOVED and uri=air.com.RainbowSquad
05-10 15:23:32.354: E/Drive.UninstallOperation(2728): Package still installed air.com.RainbowSquad
05-10 15:23:34.880: D/Wear_Controller(3062): Received broadcast action=android.intent.action.PACKAGE_REMOVED and uri=air.com.RainbowSquad
05-10 15:23:34.902: E/Drive.UninstallOperation(3062): Package still installed air.com.RainbowSquad
05-10 15:23:36.270: I/Finsky(32704): [1] com.google.android.finsky.utils.PermissionPolicies$PermissionPolicyService.onStartCommand(18): post-install permissions check for air.com.RainbowSquad
05-10 15:23:38.039: D/Wear_Controller(21277): Received broadcast action=android.intent.action.PACKAGE_ADDED and uri=air.com.RainbowSquad
05-10 15:23:40.651: I/Finsky(2684): [1] com.google.android.finsky.utils.PermissionPolicies$PermissionPolicyService.onStartCommand(18): post-install permissions check for air.com.RainbowSquad
05-10 15:23:41.877: D/Wear_Controller(2728): Received broadcast action=android.intent.action.PACKAGE_ADDED and uri=air.com.RainbowSquad
05-10 15:23:43.704: I/Finsky(2930): [1] com.google.android.finsky.utils.PermissionPolicies$PermissionPolicyService.onStartCommand(18): post-install permissions check for air.com.RainbowSquad
05-10 15:23:45.172: D/Wear_Controller(3808): Received broadcast action=android.intent.action.PACKAGE_ADDED and uri=air.com.RainbowSquad
05-10 15:24:00.913: I/ActivityManager(609): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=air.com.RainbowSquad/.AppEntry bnds=[24,180][216,372] (has extras)} from uid 10013 on display 0
05-10 15:24:00.948: I/ActivityManager(609): Start proc 4739:air.com.RainbowSquad/u0a363 for activity air.com.RainbowSquad/.AppEntry
05-10 15:24:01.101: I/FA(4739):   adb shell setprop debug.firebase.analytics.app air.com.RainbowSquad
05-10 15:24:06.869: I/ActivityManager(609): Displayed air.com.RainbowSquad/.AppEntry: +5s931ms
05-10 15:24:11.427: I/Finsky(4781): [22824] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:24:11.431: I/Finsky(4781): [22825] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:24:11.496: I/Finsky(4781): [22824] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:24:11.500: I/Finsky(4781): [22825] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:24:11.521: I/Finsky(4781): [22824] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:24:11.524: I/Finsky(4781): [22825] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:24:11.544: I/Finsky(4781): [22824] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:24:11.547: I/Finsky(4781): [22825] com.google.android.finsky.billing.iab.y.a(27): air.com.RainbowSquad: Account determined from installer data - [jJNwc0kr_5nsAusbH7SCwf69nWI]
05-10 15:24:13.089: I/b(4739): ##### Serializing blacklist ad id to /data/user/0/air.com.RainbowSquad/cache/.chartboost/.adId
05-10 15:24:13.108: D/b(4739): ##### Before html injection file path /data/user/0/air.com.RainbowSquad/cache/.chartboost/html/2cbef49f36abd8d889280345535c53fb5037c9e1-2107639096
05-10 15:24:13.573: I/b(4739): ##### Serializing blacklist ad id to /data/user/0/air.com.RainbowSquad/cache/.chartboost/.adId

**Comment by myflashlab** [comment]
Will you also post your manifest? One thing that might be happening here is that you are not asking for the ```<uses-permission android:name="android.permission.GET_ACCOUNTS" />``` permission?

**Comment by myflashlab** [comment]
Just released V4.0.0 of this ANE and this ANE now uses the [GoogleSignIn ANE](https://github.com/myflashlab/GoogleSignIn-ANE) to take care of the authentication. Update yourself and make sure to read the new Wiki to see how you should do things. If any problem, please open a new issue because authentication inside the GameServices ANE is no longer relevant.
